src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -450,7 +450,7 @@ private void markAsUnread() {
         Optional<GuiElement> item = listView.getSelectedItem();
         if (item.isPresent()) {
             TurboIssue issue = item.get().getIssue();
-            issue.markAsUnread(ui.prefs);
+            issue.markAsUnread(UI.prefs);
```

Comment:
Don't change things that's not related to this PR/issue. And I think `ui` is the correct one.